### PR TITLE
New england brew

### PR DESCRIPTION
**Pull request by ChrisWeissmann**
Adding template and first draft of new england IPA

### CODE REVIEW

**Review by c7h — CHANGES_REQUESTED** [review]
Pretty much ok but some changes expected. Let's add TODOS as comments in the future.

File: _posts/2020-07-03-NewEngland.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- 3.6 kg Pilsener extra light
+- 800 g Wheat malt light
+- 666 g Carared malt
+- 470 g Vienna malt
+- 333 g Oats
+
+### Hops / flavoring
+
+Total boiling time is xx minutes from hot break.
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md:39]: When else do you start the boiling time? it is important to state the boiling time

File: _posts/2020-07-03-NewEngland.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+~20L Mash -> roughly calculated by measuring width and hight of pot
+
+10L Sparge
+
+We used Berlin tap water which is considered medium hard to hard (14 °dH).
+
+### Fermentables
+
+Todays fermentable were eyeballed pretty hard. We bought the ingredients for a 30L mash in but only want 20L due to brew pot capacity. Therefor we estimated 2/3 of our ingredients.
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md:29]: It makes it interesting ;)

File: _posts/2020-07-03-NewEngland.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+title: "New England IPA"
+date: 2020-07-02 00:09:10 +0100
+categories: recipes
+---
+
+# Nerdkiez WIP Beer!
+
+First time brewed: 03 July 2020
+
+**TODO: Add Image HERE**
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md]: WIP image added.

File: _posts/2020-07-03-NewEngland.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+| Mash name     | Temperature | Time in rest |
+| ------------- | ----------- | ------------ |
+| Mash in       | 50°C        |              |
+| Combined Rest | 67°C        | 60 min       |
+| Mash out      | 78°C        |              |
+
+## Fermentation
+
+US S5 and S33
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md]: Added

File: _posts/2020-07-03-NewEngland.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Fermentation takes about **x** to **x** days at **x**°C
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md]: Added

File: _posts/2020-07-03-NewEngland.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+## Brewer's tip
+
+Science is more art than science.
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md:72]: Added with link to youtube

File: _posts/2020-07-03-NewEngland.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+# Nerdkiez WIP Beer!
+
+First time brewed: 03 July 2020
+
+**TODO: Add Image HERE**
+
+## Description
+
+**TODO: Description of flavor and color**
+
```
[the remark this answers is not in the document]
Comment (ChrisWeissmann) [reply _posts/2020-07-03-NewEngland.md]: added what the aim was.

**Pull request merged**